SafeNeighbourhood/Pages/Index.cshtml

```diff
@@ -10,7 +10,7 @@
     <h1 class="display-4">Welcome</h1>
     <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
   
-    <table>
+    <table class="table"> <!-- to improve UI-->
```

Comment:
To increase readability of the UI

Reply:
The discussion here in the pull request is good, but remove this from the HTML... it's clutter.
<!-- to improve UI-->